### DISCUSSION TITLE
Install vaultwarden

### DISCUSSION THREAD

**Discussion by maasnet** [dani-garcia/vaultwarden#2009] [General]
When I install vaultwarden with Portainer (vaultwarden/server:latest) I get this error:
**No such image**

**Comment by BlackDex** [comment]
Could be that docker hub is not the default maybe? Or that you need to provide the correct registry?
It's not something we can fix. All the images are available from the docker hub registry.
https://hub.docker.com/r/vaultwarden/server

**Comment by maasnet** [comment]
When I try it in docker I get the following error:
sudo docker pull vaultwarden/server
Using default tag: latest
Error response from daemon: Get "https://registry-1.docker.io/v2/": dial tcp: lookup registry-1.docker.io on 192.168.0.128:53: read udp 192.168.0.155:40826->192.168.0.128:53: i/o timeout

**Reply by BlackDex** [reply]
`192.168.0.0` are local/private ip address ranges. So it looks like something is overriding that DNS.
Looks like something on your host which blocks this, or has changed the DNS. Also, try to use an other image like `portainer/portainer-ce:alpine` for example, and see if that works.

**Comment by maasnet** [comment]
I reinstalled everything on my raspberry and now I have no more problems.
Thanks for the feedback
